tests/end_to_end/features/steps/autovariants.py

```
@@ -101,19 +126,31 @@
-def step_impl(context, name):
-    @context.spark.df_transformation(
-        name=name,
-        inputs=[context.transactions],
-    )
-    def some_transformation(df):
-        """modified transactions"""
-        from pyspark.sql.functions import lit
-
-        df = df.withColumn("new_column", lit("hi.world"))
-        return df
+def step_impl(context, name, tf_type):
```

Ruff (0.8.2)

Redefinition of unused `step_impl` from line 113

(F811)